### PR TITLE
Add some basic initial tests

### PR DESCRIPTION
**Pull request by muffinresearch**
Relates to #584

This adds jest and some very simple tests looking at the frontmatter.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "dompurify": "2.0.12",
     "foundation-sites": "6.6.3",
+    "glob": "^7.1.6",
```
Comment (willdurand) [inline package.json:18]: Should we get rid of the carets in the added deps?

File: tests/frontmatter.test.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    let fm = matter.read(mdFile).data;
+
+    // Ignore the homepage for title and description as it's special.
+    if (!mdFile.match(/index\.md/)) {
+      it(`${mdFile} should have description under 70 chars`, () => {
+        expect(fm.title).toBeDefined();
+        expect(fm.title.length).toBeLessThanOrEqual(70);
+      });
+
+      // Skipping to start with because we don't have this yet.
```
Comment (willdurand) [inline tests/frontmatter.test.js:15]: Then maybe remove it?

File: tests/frontmatter.test.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+const glob = require('glob');
+const matter = require('gray-matter');
+
+describe('frontmatter tests', () => {
+  for (const mdFile of glob.sync('src/content/**/*.md')) {
```
Comment (willdurand) [inline tests/frontmatter.test.js:5]: Did you try to use jest `.each` instead?

Comment (muffinresearch) [reply tests/frontmatter.test.js:5]: I did look at it, but it's not a case of re-using the same data each time, the expected vs input is different. I think overall this approach is more direct and easier to understand.

File: tests/frontmatter.test.js (hunk 2 above)
Comment (muffinresearch) [reply tests/frontmatter.test.js:15]: Yep, fair enough - I'll add this example to the issue for adding descriptions as a guide.

File: tests/frontmatter.test.js (hunk 3 above)
Comment (willdurand) [reply tests/frontmatter.test.js:5]: ok yep, works for me in any case 👍

**Comment by muffinresearch** [comment]
@willdurand Thanks for the review!

**Pull request merged**